IconSDK.Tests/TestRPC.cs

```diff
@@ -13,8 +14,22 @@ public class TestRPC
         public async Task Test_GetBalance()
         {
             var GetBalance = new GetBalance(API_URL);
+
+            // case 0: Successfull calls
+            // Get balance from EOA or score successfully
             var balance = await GetBalance.Invoke("hx0000000000000000000000000000000000000000");
-            Console.WriteLine($"Treasury : {balance.ToString()}");
+            Assert.IsInstanceOf<System.Numerics.BigInteger>(balance);
+            // No prefix
+            var balance2 = await GetBalance.Invoke("0000000000000000000000000000000000000000");
```

Comment:
I'm actually surprised that it is returns the balance correctly. I though icon-service or loopchain nodes would have rejected such address without any prefix, but they do not (also confirmed with the icon-sdk-python)

Reply:
As you know an address consists of bytes. So I decided that I make it casted to 'ExternalAddress' object implicitly if its format is fit with hex string. The SDK treats addresses as 'ExternalAddress' or 'ContractAddress' objects internally. 
But you can see exceptions when you use invalid hex string or wrong sized hex string. 
Aside from this I know exceptions are not specified. All exceptions are default exception(System.Exception). I will modify it to custom exception someday.